dojo/tools/securecodebox/parser.py

```diff
@@ -0,0 +1,89 @@
+import hashlib
+import json
+from urllib.parse import urlparse
```

Comment:
We want every parser to use `hyperlink` package. We need to rewrite the part that manage endpoints to leverage `hyperlink.parse()` instead

Reply:
Done